lms/templates/instructor/instructor_dashboard_2/cohort-discussions-inline.underscore

```diff
@@ -0,0 +1,30 @@
+<hr class="divider divider-lv1" />
+
+<form action="" method="post" id="cohort-inline-discussions-form" class="cohort-inline-discussions-form">
+    <div class="wrapper cohort-management-supplemental">
+        <div class="form-fields">
+            <div class="form-field">
+                <div class="inline-discussion-topics">
+                    <h3 class="subsection-title"><%- gettext('Content-Specific Discussion Topics') %></h3>
+                    <p><%- gettext('Specify whether content-specific discussion topics are divided by cohort.') %></p>
+                    <div class="always_cohort_inline_discussions">
+                        <input type="radio" name="inline" class="check-all-inline-discussions" <%- alwaysCohortInlineDiscussions ? 'checked="checked"' : '' %>/>
+                        <span class="all-inline-discussions"><%= interpolate('Always cohort content-specific discussion topics') %></span>
```

Comment:
You should also use ``<%-`` so that translations with Unicode characters will be escaped correctly.